doc/man1/openssl-verification-options.pod

```diff
@@ -121,7 +137,8 @@ if all of the following conditions hold:
 
 =item *
 
-Its subject name matches the issuer name of the subject certificate.
+Its subject Distinguished Name (DN)
+matches the issuer DN of the subject certificate.
```

Comment:
Below it states that the subject info in the issuer certificate need not be present even if specified in the subject certificate.  I suspect this may not be true. Please check.

Reply:
What exactly were you referring to here?
Already in spring 2023 I did not find it, and also when trying now, I do not see what you meant.
Of course there is no statement that the subject of a cert does not need to be there, since it is required in the X.509 format already syntactically.